### ISSUE TITLE
User Manual and Examples

### ISSUE THREAD

**Issue by Minnielam** [PAHFIT/pahfit#104]
Hi all,

I am a new user for pahfit, would it be possible to add some examples for quick overview? Many thanks.

Man I

**Comment by karllark** [comment]
Hi - can you be a bit more specific of the kind of examples you would like?  What are you wanting to do with pahfit is some detail so we can provide an appropriate example.

**Comment by jdtsmith** [comment]
We're in a pretty intensive development phase right now, we should probably add something to the main page to that effect.  We do intend to develop some example-howto's.  In the meantime, the IDL version of PAHFIT is still working well with Spitzer/IRS spectra; see http://tir.astro.utoledo.edu/jdsmith/research/pahfit.php for more info.

**Comment by Minnielam** [comment]
Thanks a lot for @karllark and @jdtsmith' reply.  I think it would be useful to have an example which shows 'how to run pahfit', or a test case which could test the installation status.

**Comment by karllark** [comment]
Have you checked out the docs?  https://pahfit.readthedocs.io/en/latest/  There is info on how to run pahfit and plot the result.  Any comments you have on these docs are welcome.

**Comment by Minnielam** [comment]
Hi @karllark , thanks a lot for your quick reaction. 

In fact, I read this document, and found the running script run_pahfit was not on my file. That's the reason for the initial question. 
I imported pahfit package, which was successful. Then, I ran the example which was shown in the document - 'How to fit a spectrum', and that was fail. The data file 'M101_Nucleus_irs.ipac'  was there at ~/pahfit/data/, and the problem was missing run_pahfit.py (or probably in another name that I did not notice). Thus, I would like to run a simple example or probably a test case to check whether the installation is not complete. 

Besides, I installed pahfit by using pip installation.
